src/pages/Filing/UpdateFinancialProfile/types.ts

```diff
@@ -91,3 +92,42 @@ export const UpdateInstitutionSchema = institutionDetailsApiTypeSchema
   );
 
 export type UpdateInstitutionType = z.infer<typeof UpdateInstitutionSchema>;
+
+// TODO: Find a better way to reuse/extend the UpdateInstitutionSchema for UpdateInstitutionTypeSchema
```

Comment:
An interesting type here for sure! I think you could maybe use a `.pick()` to maybe just grab what you need, so you'd at least not have to redefine:
```js
    sbl_institution_types: z.boolean().optional().array(),
    sbl_institution_types_other: z.string().optional(),
```

I might be wrong about that, but take a peek?

Reply:
Thank for the rec but I'm not finding success with the `pick` approach, and just found a bug in [Update your FI profile](https://github.com/cfpb/sbl-frontend/issues/513), so will revisit this investigation as part of my efforts there.

Reply:
Can we make a quick ticket then for this? Otherwise, we'll have a secret duplicate if we ever change one of these types?

Reply:
@billhimmelsbach https://github.com/cfpb/sbl-frontend/issues/522